src/index.ts

```diff
@@ -19,10 +19,12 @@ export interface IMetadataComponentContent {
   description: string;
   imageHeader: string;
   less: string;
+  lessAriaLabelTemplate;
```

Comment:
Looks like the missing type you fixed in the other PR didn't carry over to this one.